crypto_sdk/src/contracts/Darilka.sol

```diff
@@ -70,7 +87,7 @@ contract Darilka {
 
     function bookTransfer(address receiver, address nftContract, uint256 tokenId) payable public isOwner {
         bookedTransfer[keccak256(abi.encodePacked(nftContract, tokenId))] = receiver;
-        payable(receiver).transfer(comission / 2);
+        payable(receiver).transfer(commission / 2);
```

Comment:
do gift receivers really need 1/2 of the commission? He needs only for gas transfer